doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+these offered topics, the middleware daemon, called ``Roudi``, must be running.
+
+For further information how iceoryx can be used see the [examples](../iceoryx_examples/README.md). The
+[conceptual-guide.md](../../conceptual-guide.md) provides additional information about the ``Shared Memory
+communication`` that lies at the heart of iceoryx.
+
+We now briefly define the main entities of an iceoryx system before showing how they are created and used by the
+iceoryx API.
+### Roudi
+
+Roudi is an abbreviation for **Rou**ting and **Di**scovery. Roudi takes care of the
+communication setup but does not actually participate in the communication between the publisher and the subscriber.
+RouDi can be thought of as the switchboard operator of iceoryx. One of his other major tasks is the setup of the
+shared memory, which the applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `iox-roudi --help`.
+
+### Runtime
+
+Each application which wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with Roudi. Only one runtime object per user process is allowed.
+
+To do so, the following lines of code are required
+
+    #include "iceoryx_posh/runtime/posh_runtime.hpp"
+
+    iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
+
+### Topics
+
+A topic in iceoryx defines the data to be transmitted and is uniquely identified by three string identifiers.
+
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
+
+A triple consisting of such strings is called a ``service description``. In Autosar terminology these three
+identifiers are called ``Service``, ``Instance`` and ``Event`` respectively.
+
+Two topics are considered matching if all these three strings are element-wise equal, i.e. group, instance and topic
+names are the same for both of them.
+
+This means the group and instance identifier can be ignored to create different topics. They will be needed for
+advanced filtering functionality in the future.
+
+The data type of the topic can be an arbitrary C++ class, struct or plain old data type.
+
+### Publisher
+A publisher is tied to a topic and needs a service description to be constructed. If it is typed one needs to
+additionally specify the data type as a template parameter. Otherwise publisher is only aware of raw memory and the
+user has to take care that it is interpreted correctly.
+
+Once it has offered its topic, it is able to publish (send) data of the specific type. Note that it is possible to
+have multiple publishers for the same topic.
```

Comment:
Maybe we should mention the n:m switch here? Or later in `installation.md`? What do you think @dkroenke ?

Reply:
I leave this open for now.

Reply:
Can you add an todo?

Reply:
I thought n:m is the default case. Were should the todo be placed? Until we do not know how to configure it, we should not mention it here. I also feel it belongs to installation if it is based on a cmake flag.

Reply:
Just mentioned it high-level